packages/core/src/baseclient.ts

```diff
@@ -653,6 +654,29 @@ export abstract class BaseClient<O extends ClientOptions> implements Client<O> {
           throw new SentryError('`beforeSend` returned `null`, will not send event.', 'log');
         }
 
+        const transactionInfo = processedEvent.transaction_info;
+        if (
+          processedEvent.type === 'transaction' &&
+          transactionInfo &&
+          processedEvent.transaction &&
+          processedEvent.transaction !== event.transaction
+        ) {
+          const source = 'custom';
+          event.transaction_info = {
+            ...transactionInfo,
+            source,
+            name_changes: [
+              ...transactionInfo.name_changes,
+              {
+                name: processedEvent.transaction,
```

Comment:
Should this still be here?

Reply:
Nope - didn't get removed as part of the [3ef0b80](https://github.com/getsentry/sentry-javascript/pull/5709/commits/3ef0b80739218140e45e8ecfb13e63412800227d) accidently